install-web-ui.sh

```diff
@@ -1,16 +1,22 @@
 # Copyright 2015-2017, Dell EMC, Inc.
 
 cd ./static
-
 rm -rf web-ui
-curl --fail -L -o on-web-ui-gh-pages.zip https://github.com/RackHD/on-web-ui/archive/gh-pages.zip
+version=$(curl https://raw.githubusercontent.com/rackhd/rackhd-ui-2.0/version/version)
+curl --fail -L -o on-web-ui-gh-pages.zip https://github.com/rackhd/rackhd-ui-2.0/archive/${version}.zip
 
 if [ $? -ne 0 ]; then
     echo 'failed to download, retrying'
-    curl --fail -L -o on-web-ui-gh-pages.zip https://github.com/RackHD/on-web-ui/archive/gh-pages.zip \
+    curl --fail -L -o on-web-ui-gh-pages.zip https://github.com/rackhd/rackhd-ui-2.0/archive/${version}.zip \
```

Comment:
We should use CI/CD pipeline to do build work and upload built files as .zip package.